app/views/works/show.html.erb

```diff
@@ -41,11 +41,9 @@
 
 <!--chapter content-->
     <div id="chapters" role="article">
-      <% if @chapters && @chapters.size > 1 %>
+      <% if @chapters %>
         <% for chapter in @chapters %>
-          <% if chapter.posted? || is_author_of?(chapter) %>
-            <%= render :partial => 'chapters/chapter', :locals => {:chapter => chapter, :hide_work_info => true} %>
-          <% end %>
+          <%= render 'chapters/chapter', chapter: chapter, hide_work_info: true %>
```

Comment:
What is `hide_work_info`? This seems to be the only reference to it I can find.

Reply:
I'm not sure. It looks like the last reference to it in the chapter view was removed in [this commit from 2008](https://github.com/otwcode/otwarchive/commit/0e3138342d2d5ad65ab67339f8f4430d6f6da4ea#diff-7bafc42314ce6e39760829569596a4c5). Do you want me to remove it?